fire/core.py

```diff
@@ -281,6 +282,45 @@ def _PrintResult(component_trace, verbose=False, serialize=None):
     Display(output, out=sys.stdout)
 
 
+def _GetProgName(name, main=None):
```

Comment:
In the current PR I don't think the main arg is passed anywhere (other than the test). Is a file missing?

Reply:
It is only passed during testing.